homeassistant/components/poolstation/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Support for Rituals Perfume Genie sensors."""
+from __future__ import annotations
+
+from pypoolstation import Pool
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DEVICE_CLASS_TEMPERATURE, PERCENTAGE, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import PoolstationDataUpdateCoordinator
+from .const import COORDINATORS, DEVICES, DOMAIN
+from .entity import PoolEntity
+
+PH_SUFFIX = " PH"
+TEMPERATURE_SUFFIX = " Temperature"
+SALT_SUFFIX = " Salt concentration"
+ELECTROLYSIS_SUFFIX = " Electrolysis"
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the poolstation sensors."""
+    stations = hass.data[DOMAIN][config_entry.entry_id][DEVICES]
+    coordinators = hass.data[DOMAIN][config_entry.entry_id][COORDINATORS]
+    entities: list[PoolEntity] = []
+    for pool_id, pool in stations.items():
+        coordinator = coordinators[pool_id]
+        entities.append(PoolPhSensor(pool, coordinator))
+        entities.append(PoolTemperatureSensor(pool, coordinator))
+        entities.append(PoolSaltConcentrationSensor(pool, coordinator))
+        entities.append(PoolElectrolysisSensor(pool, coordinator))
+
+    async_add_entities(entities)
+
+
+class PoolPhSensor(PoolEntity, SensorEntity):
+    """Representation of a pool's PH sensor."""
+
+    _attr_icon = "mdi:flask"
```

Comment:
Have you seen the recently added `mdi:ph` icon?
https://materialdesignicons.com/icon/ph

Reply:
I did and it's what I tried first, it's not available in the version of the icons bundled with HA right now. I believe HA uses 5.X and it was added in 6.X